src/test/java/com/microsoft/sqlserver/jdbc/resultset/ResultSetTest.java

```diff
@@ -636,11 +693,10 @@ public void testResultSetFetchBufferSqlErrorState() throws Exception {
     @Test
     public void testResultSetClientCursorInitializerSqlErrorState() throws Exception {
```

Comment:
No exception is ever thrown
```suggestion
    public void testResultSetClientCursorInitializerSqlErrorState() {
```